strongbox-parent/pom.xml

```diff
@@ -63,7 +63,7 @@
         <version.spring>4.3.6.RELEASE</version.spring>
         <version.spring.security>4.2.1.RELEASE</version.spring.security>
         <version.spring.data>0.10.0-SNAPSHOT</version.spring.data>
-        <version.orientdb>2.2.16</version.orientdb>
+        <version.orientdb>2.2.18-SNAPSHOT</version.orientdb>
```

Comment:
Im aware of using SNAPSHOT versions in such kind of important places. Suggest to get back to stable release version.